src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -0,0 +1,38 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2020 1745 JJ Pearce Robotics. All Rights Reserved.           */
```

Comment:
If we're open design[0] I don't know if we want to mark stuff as copyright

https://www.chiefdelphi.com/t/1745-is-going-open-design-for-2020-again/368987

>For 2020, 1745 The P-51 Mustangs have decided to have all of our working design files in the “Open”; CAD, Code, sketches, math, flow charts it’s all there for anyone to read/copy. 

Perhaps we should settle on a license for our code. 

cc: @Stuart1745

Reply:
MIT? GPLv3?

Reply:
I like both of those licenses!